src/os/unix/mod.rs

```diff
@@ -19,26 +19,43 @@ pub use x11::Menu;
 
 use std::os::raw;
 
-//Differentiate between Wayland and X11 at run-time
+// Differentiate between Wayland and X11 at run-time
 pub enum Window {
     X11(x11::Window),
-    Wayland(()), //WlWindow
+    Wayland(wayland::Window),
 }
 
```

Comment:
Basically everything in this file that refers to one or the other needs to be feature-gated.